lxd/instance/drivers/driver_lxc.go

```diff
@@ -352,7 +353,8 @@ func lxcCreate(s *state.State, args db.InstanceArgs, revert *revert.Reverter) (i
 	}
 
 	d.logger.Info("Created container", log.Ctx{"ephemeral": d.ephemeral})
-	d.lifecycle("created", nil)
+	event := lifecycle.InstanceCreated.Lifecycle(d.name, d.project, d.GetLifecycleInstanceType(), nil, d.op)
```

Comment:
So here, it should be `d.state.Events.SendLifecycle(lifecycle.InstanceCreated.Event(d, nil))`